ui/app/milestone/milestone-controllers.js

```diff
@@ -21,6 +21,27 @@
 
   var module = angular.module('pnc.milestone');
 
+  module.controller('MilestoneDetailController', [
+    '$scope',
+    '$state',
+    '$stateParams',
+    'productDetail',
+    'versionDetail',
+    'milestoneDetail',
+    'distributedArtifacts',
+    'performedBuilds',
+    function($scope, $state, $stateParams, productDetail, versionDetail, milestoneDetail,
+        distributedArtifacts, performedBuilds) {
+
+      var that = this;
+      that.product = productDetail;
+      that.productVersion = versionDetail;
+      that.milestone = milestoneDetail;
+      that.distributedArtifacts = distributedArtifacts.data;
```

Comment:
Hi @pgier ,

http://localhost:9000/#/product/1/version/1/milestone/1/detail Shipped Artifacts part is empty until page is refreshed.

I think there data attribute is not resolved properly.